cas-server-support-x509/pom.xml

```diff
@@ -54,10 +54,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <artifactId>vt-crypt</artifactId>
-      <groupId>edu.vt.middleware</groupId>
-      <version>2.1.4</version>
-    </dependency>
+	  <groupId>org.cryptacular</groupId>
```

Comment:
I searched cas 4.1.0 for references of vt-crypt and I found that dependency only in cas-server-support-x509, so it is not used anywhere else.